### ISSUE TITLE
Error building / go get Windows 32 bit OS

### ISSUE THREAD

**Issue by scohen28** [mattn/go-sqlite3#166]
$ go get github.com/mattn/go-sqlite3
# github.com/mattn/go-sqlite3
<command-line>:0:14: error: conflicting types for 'localtime'
In file included from ..\..\..\github.com\mattn\go-sqlite3\sqlite3.c:14589:0:
c:\mingw\include\time.h:122:45: note: previous declaration of 'localtime' was he
re
 _CRTIMP struct tm*  __cdecl __MINGW_NOTHROW localtime (const time_t*);
                                             ^
..\..\..\github.com\mattn\go-sqlite3\sqlite3.c: In function 'osLocaltime':
..\..\..\github.com\mattn\go-sqlite3\sqlite3.c:14983:3: warning: passing argumen
t 1 of 'localtime' from incompatible pointer type [enabled by default]
   pX = localtime(t);
   ^
<command-line>:0:14: note: expected 'const __time32_t *' but argument is of type
 'time_t *'

**Comment by mattn** [comment]
What version of gcc do you use?

**Comment by scohen28** [comment]
$ mingw32-gcc.exe --version
mingw32-gcc.exe (GCC) 4.8.1

**Comment by scohen28** [comment]
Think I've got it solved.  I had installed MinGW on my windows VM just using the generic Windows Installer - up until the latest go-sqlite3 updates guess everything was fine.  Looking at the MinGW include directory the time.h file was dated 9/2013.  In the MinGW installer I selected additional packages (base, gcc and one other), applied the update and received a new time.h file, dated 12.2014 - now go-sqlite3 compiles as before - so looks like the base MnGW install doesn't have the proper includes required - fyi
